rom/dev/src/flow/cold_reset/idev_id.rs

```diff
@@ -36,6 +40,15 @@ const DOE_IV: Array4x4 = Array4xN::<4, 16>([0xfb10365b, 0xa1179741, 0xfba193a1,
 /// Maximum Certificate Signing Request Size
 const MAX_CSR_SIZE: usize = 512;
 
+const DUAL_CERT_MODE: bool = true;
+
+struct MLdsaKeyVaultInfo {
```

Comment:
Nit: I think we have been using `Mldsa` in most other places (with the `L` lowercase), though for the public key we use `MlDsa`. It would be nice if we were consistent.